.travis.yml

```diff
@@ -1,9 +1,5 @@
-sudo: false
-
-branches:
-  only:
-    - master
-    - /^v\d+\.\d+\.\d+$/
+sudo: true
+dist: xenial
```

Comment:
We shouldn't update to xenial until April of 2019. Ubuntu 14 is our minimum support until LTS ends.

Reply:
Really? Xenial seems to be pretty standard now.  Is it possible that trusty would work and xenial would fail? Which would be more likely.

Reply:
It's more likely that xenial would work and trusty would fail. My main concern is that libcurl dependency for proxy support being dynamically linked.